hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassCluster.java

```diff
@@ -64,21 +76,111 @@
     private static final String ALERTS_CASSANDRA_READ_TIMEOUT = "hawkular-alerts.cassandra-read-timeout";
     private static final String ALERTS_CASSANDRA_READ_TIMEOUT_ENV = "CASSANDRA_READ_TIMEOUT";
 
+    /*
+        GLOBAL OVERWRITE
```

Comment:
ops, this comment was left here.
I forgot to remove it.
This is to replace the getSession(overwrite) method with the possibility to drop/recreate the existing scheme.
I will update the comment.